src/main/webapp/app/entities/quiz-exam-exercise.ts

```diff
@@ -0,0 +1,41 @@
+import { ExerciseType, IncludedInOverallScore } from 'app/entities/exercise.model';
+import { StudentParticipation } from 'app/entities/participation/student-participation.model';
+import { InitializationState } from 'app/entities/participation/participation.model';
+import { ExamExercise } from 'app/entities/exam-exercise';
+import { QuizConfiguration } from 'app/entities/quiz/quiz-configuration.model';
+import { ExerciseGroup } from 'app/entities/exercise-group.model';
+import { QuizQuestion } from 'app/entities/quiz/quiz-question.model';
+import { QuizSubmission } from 'app/entities/quiz/quiz-submission.model';
+
+export class QuizExamExercise implements ExamExercise, QuizConfiguration {
+    public id?: number;
+    public type?: ExerciseType;
+    public studentParticipations?: StudentParticipation[];
+    public navigationTitle?: string;
+    public overviewTitle?: string;
+    public exerciseGroup?: ExerciseGroup;
+    public quizQuestions?: QuizQuestion[];
+    public randomizeQuestionOrder?: boolean;
+    public title?: string;
+    public maxPoints?: number;
+    public includedInOverallScore?: IncludedInOverallScore;
+
+    constructor() {
+        this.id = 0;
+        this.type = ExerciseType.QUIZ;
+        const submission = new QuizSubmission();
+        submission.isSynced = true;
+        this.studentParticipations = [
+            {
+                initializationState: InitializationState.INITIALIZED,
+                submissions: [submission],
+            },
+        ];
+        this.navigationTitle = 'Quiz';
+        this.exerciseGroup = new ExerciseGroup();
+        this.exerciseGroup.title = 'Quiz Exam';
+        this.overviewTitle = 'Quiz Exam';
+        this.title = 'Quiz Exam';
+        this.includedInOverallScore = IncludedInOverallScore.INCLUDED_COMPLETELY;
```

Comment:
The values are intended and displayed during the exam conduction. But now, I refactored it a bit to use translated values instead of hard-coded values.